### ISSUE TITLE
Buttons &:active and &:focus default styles

### ISSUE THREAD

**Issue by nastiatikk** [mobify/stencil#138]
Regarding [Issue-66](https://github.com/mobify/stencil/issues/66) we need to decide if default `&:focus` and `&:active` states are something that we want to have for buttons

@ry5n @avelinet @jeffkamo @yourpalsonja @cole-sanderson @mlworks @kpeatt

**Comment by jeffkamo** [comment]
I agree with Ryan that we don't need hover. However, I suspect we should still have some form of `:active` and `:focus` for a11y reasons, mainly: screen reader user agents.

If we're not sure (which we aren't... at least I'm not 100% sure) then we should test VoiceOver and the Android equivelant.

It also might just be a matter of having at least `outline` on focus/active, ala http://outlinenone.com/.
